frontend/src/Constants.js

```diff
@@ -0,0 +1,9 @@
+const prod = {
+  WS_URL: "ws://example.com"
```

Comment:
In prod we want the websocket to connect to the same host and port that the site is served from, so: `"ws://" + window.location.host`